tests/rds_option_group: error with `InvalidParameterCombination: Cannot find major version`

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Affected Test(s)

<!--- Please list the affected resources and data sources. --->

* TestAccRDSOptionGroup_Option_optionSettings
* TestAccRDSOptionGroup_multipleOptions

### Example output:
```
------- Stdout: -------
=== RUN   TestAccRDSOptionGroup_multipleOptions
=== PAUSE TestAccRDSOptionGroup_multipleOptions
=== CONT  TestAccRDSOptionGroup_multipleOptions
    option_group_test.go:389: Step 1/2 error: Error running apply: exit status 1
        
        Error: Error creating DB Option Group: InvalidParameterCombination: Cannot find major version 11.2 for oracle-se
        	status code: 400, request id: 537a3d21-26d9-411f-8edc-f8edbc85240c
        
          on terraform_plugin_test.tf line 2, in resource "aws_db_option_group" "bar":
           2: resource "aws_db_option_group" "bar" {
        
        
--- FAIL: TestAccRDSOptionGroup_multipleOptions (6.02s)
```

--- a/internal/service/rds/option_group_test.go
+++ b/internal/service/rds/option_group_test.go
@@ -675,7 +675,7 @@ resource "aws_db_option_group" "bar" {
   name                     = "%s"
   option_group_description = "Test option group for terraform"
   engine_name              = "oracle-ee"
-  major_engine_version     = "11.2"
+  major_engine_version     = "12.2"
 
   option {
     option_name = "Timezone"
@@ -733,7 +733,7 @@ resource "aws_db_option_group" "bar" {
   name                     = "%s"
   option_group_description = "Test option group for terraform"
   engine_name              = "oracle-ee"
-  major_engine_version     = "11.2"
+  major_engine_version     = "12.2"
 
   option {
     option_name = "Timezone"
@@ -783,7 +783,7 @@ resource "aws_db_option_group" "bar" {
   name                     = "%[1]s"
   option_group_description = "Test option group for terraform issue 748"
   engine_name              = "oracle-ee"
-  major_engine_version     = "12.1"
+  major_engine_version     = "12.2"
 
   option {
     option_name = "OEM_AGENT"
@@ -816,15 +816,15 @@ func testAccOptionGroupMultipleOptions(r string) string {
 resource "aws_db_option_group" "bar" {
   name                     = "%s"
   option_group_description = "Test option group for terraform"
-  engine_name              = "oracle-se"
-  major_engine_version     = "11.2"
+  engine_name              = "oracle-ee"
+  major_engine_version     = "12.2"
 
   option {
-    option_name = "STATSPACK"
+    option_name = "SPATIAL"
   }
 
   option {
-    option_name = "XMLDB"
+    option_name = "STATSPACK"
   }
 }
 `, r)

